cypress/integration/contacts.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('contacts page', function() {
+  it('form send', function() {
+    cy.visit('http://localhost:8000/contacts/');
```

Comment:
Can we get it from `process` or not? Usually Gatsby starts on 8000, but this behavior can change.

Reply:
can we have two commands
```
npm tun test // that will test localhost
npm run test:prod // that will test www.halo-lab.com
```
?